network-api/networkapi/wagtailpages/templates/wagtailpages/blog_page.html

```diff
@@ -13,7 +13,15 @@
   <div class="py-4 py-md-5 col-lg-8">
     <div class="cms">
       <h1 class="h1-heading">{{ page.title }}</h1>
-      <p class="h6-heading mb-5">By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not published yet{% endif %}</p>
+      <p class="h6-heading mb-5">
+        By {{ page.author }} | {% if page.first_published_at %}{{ page.first_published_at|date:"F j, Y" }}{% else %}not
+        published yet{% endif %} |
+        {% with category=page.specific.category.first %}
+        {% if category %}
+        <a class="blog-detailed-category" href="/blog/category/{{category.slug}}">{{ category }}</a>
```

Comment:
we probably shouldn't be hardcoding parts of this URL, we should be able to ask wagtail for the URL we need instead.

Reply:
```suggestion
        <a class="blog-detailed-category" href="{% slugurl 'blog' %}category/{{category.slug}}">{{ category }}</a>
```

Reply:
ideally we also don't hardcode `category` but fetch that from the BlogPageCategory model (e.g using a `fullUrl` property definition) but that's beyond the scope of this issue, really.

Reply:
@Pomax Definitely, not a problem. In `featured_blogs.html` the blog category url is hardcoded similar to what I've done here, just fyi for future fixes/updates 👍 .